assets/src/blocks/Counter/CounterFrontend.js

```diff
@@ -14,7 +14,9 @@ export class CounterFrontend extends Component {
 
   componentDidMount() {
     // Calculate completed and remaining values depending on props
-    this.calculateRemaining();
+    let counter = this;
+    counter.calculateRemaining();
+    window.addEventListener('updateCounter', function (e) { counter.calculateRemaining(); }, false);
```

Comment:
The `e` param is unused, it can probably be removed? 🤔 Maybe it would also be handy to add a comment to explain what is happening here 🙂

Reply:
I will remove the parameter for now :)
I am going to need and re-add it later though, because through the e.detail parameter it's possible to pass the new unique_count and that the API doesn't need to be called twice (update&read).

Will add a comment :)

Reply:
Done :)